src/pages/Faq/components/FaqEntry.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const FaqEntry = ({ data }) => {
+  console.log(data);
+  return (
+    <div className="pb-2 flex flex-wrap justify-left content-center rounded mx-auto my-5 overflow-hidden w-5/6 lg:w-1/2 ">
+      <h2 className="text-orange">{data.question}</h2>
+      <p className="text-white">{data.answer}</p>
+    </div>
+  );
```

Comment:
I think it might be nice to have a listing style similar to the way we display PR results. You may want to wait until https://github.com/jenkoian/hacktoberfest-checker/pull/377 is merged though to use that styling (will be merged very soon maybe even before you read this!)